yarn.lock

```diff
@@ -2963,15 +2963,6 @@ anymatch@^3.0.3, anymatch@~3.1.2:
     normalize-path "^3.0.0"
     picomatch "^2.0.4"
 
-"aphrodite@git+https://github.com/learningequality/aphrodite.git":
```

Comment:
It seems like dependabot was responsible for adding this - I am not sure why it does!

Reply:
Yes, I have the sense that we had removed this a couple of times already :sweat_smile:/